api/.pipeline/templates/api.dc.yaml

```diff
@@ -336,6 +346,13 @@ objects:
                   value: ${KEYCLOAK_API_HOST}
                 - name: KEYCLOAK_API_ENVIRONMENT
                   value: ${KEYCLOAK_API_ENVIRONMENT}
+                # Request limits
+                # - name: MAX_REQ_BODY_SIZE
+                #   value: ${MAX_REQ_BODY_SIZE}
```

Comment:
Commented out

Reply:
Intentional, just testing it fails when it's missing an env value

Reply:
Oh weird, I thought I wrote my second message before you commented, but I guess I didnt refresh or something